src/Windows/dll/dcap_provider.vcxproj

```diff
@@ -15,7 +15,7 @@
     <ProjectGuid>{AAA37DF8-87A5-40E9-B0DF-8919D5E92533}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
-    <!-- The library name is dicatated by Intel. They look, specifically, for a dll
+    <!-- The library name is dicatated by Intel. They look, specifically, for a dll

```

Comment:
Were these changes intentional or Visual Studios generated? If not can you please revert them?

Reply:
These were generated by Visual Studio.

Reply:
Can we revert these changes then?

Reply:
Honestly, I don't think we should since the next time someone opens the project using VS this changes will apply again.

Reply:
Outside of the explicitly commented out code, the rest of it seems to be spacing related which is fine for now. Moving forward we do expect all contributors to have VS beautification settings to be turned off.

Reply:
If that is the case we should clearly document this requirement.